### ISSUE TITLE
Deleting last app in app service plan deletes plan, no way to override

### ISSUE THREAD

**Issue by m-mertz** [Azure/azure-libraries-for-net#466]
Deleting the last app in an app service plan also deletes the plan. This behavior is consistent throughout portal and other surfaces. But in portal or cli, for example, it can be overridden, see e.g. https://github.com/Azure/azure-cli/issues/4497 (--keep-empty-plan).

In Fluent SDK, there doesn't seem to be a way to override this so that the app service plan stays around. Relevant methods (not complete list):

```csharp
appServiceManager.WebApps.DeleteByIdAsync
appServiceManager.WebApps.DeleteByResourceGroupAsync
appServiceManager.FunctionApps.DeleteByIdAsync
appServiceManager.FunctionApps.DeleteByResourceGroupAsync
```

**Comment by ppittle** [comment]
Any updates on this?

**Comment by weidongxu-microsoft** [comment]
Fixed in 1.27. To be released on Oct. 11th.

Interface:
https://github.com/Azure/azure-libraries-for-net/blob/master/src/ResourceManagement/AppService/Domain/IWebApps.cs

Sample code
`appServiceManager.WebApps.DeleteById(webApp2.Id, null, false);`
